plugins/flytekit-vscode/flytekitplugins/vscode/decorator.py

```diff
@@ -35,13 +59,56 @@ def execute_command(cmd):
     logger.info(f"stderr: {stderr}")
 
 
-def download_file(url, target_dir="."):
+def exit_handler(
+    max_idle_seconds: int = 180,
+    child_process: multiprocessing.Process = None,
+    post_execute: Optional[Callable] = None
+):
+    """
+    Check the modified time of ~/.local/share/code-server/heartbeat.
+    If it is older than max_idle_second seconds, kill the container.
```

Comment:
IMO saying "container" here is more precise because since the purpose of exit_hanlder is to shutdown the container

Reply:
+1, the whole container / resources will be recycled instead of code server. (e.g., you can't ssh to the container again after this process is terminated)